techniques/ios/MASTG-TECH-0138.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Convert Plist Files to JSON
+platform: ios
+---
+
+You can convert Plist files, such as `Info.plist` or `PrivacyInfo.xcprivacy` (also a Plist file despite the extension), to JSON format for easier readability and analysis.
+
+### Using plutil
+
+You can use @MASTG-TOOL-0062 to convert `PrivacyInfo.xcprivacy` files to JSON format, making them easier to read and analyze.
+
+```console
+plutil -convert json -o - SocialApp.app/PrivacyInfo.xcprivacy > PrivacyInfo.xcprivacy.json
```

Comment:
Add cat ... ?

Reply:
Not needed, it works like that with `>` to write to file